src/main/java/org/kitteh/tim/Tim.java

```diff
@@ -35,37 +34,33 @@
 import org.spongepowered.api.command.args.GenericArguments;
 import org.spongepowered.api.command.spec.CommandSpec;
 import org.spongepowered.api.data.key.Keys;
-import org.spongepowered.api.data.meta.ItemEnchantment;
 import org.spongepowered.api.data.type.HandTypes;
 import org.spongepowered.api.entity.living.player.Player;
 import org.spongepowered.api.event.Listener;
-import org.spongepowered.api.event.game.state.GamePostInitializationEvent;
 import org.spongepowered.api.event.game.state.GameStartingServerEvent;
-import org.spongepowered.api.item.Enchantment;
+import org.spongepowered.api.item.enchantment.Enchantment;
+import org.spongepowered.api.item.enchantment.EnchantmentType;
 import org.spongepowered.api.item.inventory.ItemStack;
 import org.spongepowered.api.plugin.Plugin;
 import org.spongepowered.api.text.Text;
 import org.spongepowered.api.text.action.TextActions;
 import org.spongepowered.api.text.format.TextColors;
 
-import java.util.Collections;
 import java.util.LinkedList;
 import java.util.List;
-import java.util.Map;
-import java.util.function.Function;
-import java.util.stream.Collector;
 
 /**
  * I... am an enchanter.
  */
-@Plugin(id = "tim", name = "Tim the Enchanter", version = "4.0.3-for-5.0")
-public class Tim {
-    public static final String COMMAND_ARG_ENCHANTMENT = "enchantment";
-    public static final String COMMAND_ARG_LEVEL = "level";
+@Plugin(id = "tim", name = "Tim the Enchanter", version = "4.1.0-for-7.0")
+public final class Tim {
 
-    public static final String PERMISSION_ENCHANT = "enchanter.enchant";
+    private static final String COMMAND_ARG_ENCHANTMENT = "enchantment";
+    private static final String COMMAND_ARG_LEVEL = "level";
 
-    public static final List<String> QUOTES = ImmutableList.of(
+    private static final String PERMISSION_ENCHANT = "enchanter.enchant";
+
+    private static final List<String> QUOTES = ImmutableList.of(
```

Comment:
These were public (and final) for visibility. Not that it's likely anyone needs anything.